src/components/header/Header.jsx

```diff
@@ -23,34 +23,33 @@ const Header = () => {
   } = useContext(GlobalContext)
 
   return (
-    <Navbar light color="light" className="header" tag="header">
+    <div className="header">
```

Comment:
Shouldn't this be `nav` or `header` element?

Reply:
[`<header>`](https://html.spec.whatwg.org/multipage/sections.html#the-header-element) is an introductory or navigation.
[`<nav>`](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) is a _section_ containing links to other sections or pages.

The toolbar does not fit into any of these definitions. It contains actions.

I think that a `<div>` is the best option here. However, I may change my mind in favor of `<header>`. Let me think about 😉